src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -535,10 +544,19 @@ function dhcpd_radvd_configure($verbose = false, $blacklist = array())
 
     file_put_contents('/var/etc/radvd.conf', $radvdconf);
 
-    if (count($radvdifs)) {
-        mwexec('/usr/local/sbin/radvd -p /var/run/radvd.pid -C /var/etc/radvd.conf -m syslog');
+    $radvd_interfaces = implode(' ',$rtadvd_interface_list);
```

Comment:
This is wrong too. You implode but later escape a la “int1 int2” which looks like one single interface that doesn’t exist vs. “int1” “int2”